opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -197,9 +197,8 @@ impl Builder {
     pub fn with_batch_exporter<T: LogExporter + 'static, R: RuntimeChannel>(
         self,
```

Comment:
This method `Builder::with_batch_exporter()` shouldn't take the runtime as generic argument anymore.

Reply:
Removed.